src/hazard/inventory.py

```diff
@@ -108,8 +111,8 @@ class HazardResource(BaseModel):
     store_netcdf_coords: bool = Field(
         False,
         description="If True, NetCDF-style coordinates are also stored, which allows XArray to read the array \
-            natively. In this case, path still points to the array; the coordinates are stored in an array group \
-                in the parent folder. That is, path should be in the form path_to_array_group/array_group/array",
+            natively. By convention, the hazard indicator data array is named 'indicator' and the path to \
+            the Zarr array is then path/indicator (not path).",
```

Comment:
So the output will have a different folder structure with this flag than without.